src/flip.sol

```diff
@@ -138,6 +167,16 @@ contract Flipper is LibNote {
         require(bid >  bids[id].bid, "Flipper/bid-not-higher");
         require(mul(bid, ONE) >= mul(beg, bids[id].bid) || bid == bids[id].tab, "Flipper/insufficient-increase");
 
+        // check for first bid only
+        if (bids[id].bid == 0) {
+            // Need to be whitelisted to make this call
+            (bytes32 val, bool has) = feed.peek();
+            if (has) {
+                uint256 par = spot.par();
+                require(bid >= rmul(wmul(rdiv(uint256(val), par), lot), cut));
```

Comment:
So I think there is an error here (haven't fully tested it) and also want to mention one consideration:

Possible error: `val` is `WAD`, `par` is `RAY`, `lot` is `WAD`, `cut` is `WAD`. `bid` should be `RAD`.

`rmul(wmul(rdiv(WAD, RAY), WAD), WAD)`

`rdiv(WAD, RAY)` is `WAD`, so `rmul(wmul(WAD, WAD), WAD)`

`wmul(WAD, WAD)` is `WAD`, so `rmul(WAD, WAD)` => 10 ^ 9 unit not 10 ^ 45 (wad).

About the consideration, I understand `cut` as the amount you would expect to `cut` to the feed price for the first `bid`. However it acts as the opposite here. I would have expected to see `sub(ONE, cut)`. Bit less efficient though.

Reply:
I actually intended `cut` to be a `ray`, the comment is wrong.
But you are correct `bid` needs to be a `rad`, not a `wad`...easy fix is to replace `rmul` with `mul`, thanks for catching!

As for naming...`cut` is far from our worst meme IMO. I can interpret it in a similar sense to its usage as a noun in phrases like "taking a cut of the profits" (here the "cut" is the amount you're getting), I think that actually matches the usage here.

Reply:
Yeah ok, makes sense!